iOS/rainbow/Controller/Leaderboard/ParticipantViewController.swift

```diff
@@ -48,9 +48,13 @@ class ParticipantViewController: UIViewController {
             if let savedFoundObjects = savedEntry.objects {
```

Comment:
If we don't show progress on this view anymore, we can get rid of the objects we use to keep track of the found objects thus far. Let's delete these, as well as the `config` and `foundObjects` array above it please.